### DISCUSSION TITLE
VirusTotal 2/73 security vendors and no sandboxes flagged this file as malicious

### DISCUSSION THREAD

**Discussion by josetetech** [cyanfish/naps2#386] [Support]
Hello, thanks in advance for the help. First of all, I want to say that I love this software, I think it is very useful on a daily basis. For some time now, all the software that I download and install on my computer I first go through the VirusTotal website to try to avoid possible viruses. The reason for opening this question is because I have downloaded the latest version of the software for Windows and it is detected by two of the analyzes as Malware. I understand that there must be some explanation, but I would be calmer if you clarified it from this thread. Thank you very much for your help. greetings. Note: if you detect bad writing in my text it is because I speak Spanish and unfortunately I do not have very good English, and I have helped myself a little with Google Translator.
![Captura de pantalla de 2024-05-10 21-16-03](https://github.com/cyanfish/naps2/assets/169490439/532827ae-347e-46f9-976f-331c1f10cb34)

**Comment by cyanfish** [comment]
Those are false positives, there's not much I can do about it as I don't have the time or resources to track down every AV vendor and try to get them to fix their incorrect results.

Another way of looking at it is that 71/73 say it is fine.

**Comment by josetetech** [comment]
okay, thank you very much for your time and effort in developing this application.
